ambry-api/src/main/java/com.github.ambry/rest/RestRequestMetrics.java

```diff
@@ -29,6 +29,7 @@
   static final String NIO_REQUEST_PROCESSING_TIME_SUFFIX = "NioRequestProcessingTimeInMs";
   static final String NIO_RESPONSE_PROCESSING_TIME_SUFFIX = "NioResponseProcessingTimeInMs";
   static final String NIO_ROUND_TRIP_TIME_SUFFIX = "NioRoundTripTimeInMs";
+  static final String NIO_FIRST_BYTE_SENT_TIME_SUFFIX = "NioFirstByteSentTimeInMs";
```

Comment:
time to first byte might be a better name - this seems to imply the abs time at which it was sent

Reply:
changed the name